app/scripts/modules/core/src/pipeline/config/stages/googleCloudBuild/GoogleCloudBuildStageForm.tsx

```diff
@@ -15,16 +15,27 @@ import {
   StageConfigField,
   yamlDocumentsToString,
   YamlEditor,
+  TextInput,
+  IGcbTrigger,
 } from '@spinnaker/core';
 
 export enum buildDefinitionSources {
   ARTIFACT = 'artifact',
   TEXT = 'text',
+  TRIGGER = 'trigger',
+}
+
+export enum triggerType {
+  BRANCH = 'branchName',
+  TAG = 'tagName',
+  COMMIT = 'commitSha',
 }
 
 interface IGoogleCloudBuildStageFormProps {
   googleCloudBuildAccounts: string[];
+  gcbTriggers: IGcbTrigger[];
   updatePipeline: (pipeline: IPipeline) => void;
+  fetchGcbTriggers: (account: String) => void;
```

Comment:
```suggestion
  fetchGcbTriggers: (account: string) => void;
```

Reply:
Just one tiny change and this is ready to merge!

Reply:
Done... ! =)